app/src/test/kotlin/com/exallium/todoapp/database/IdFactoryImplTest.kt

```diff
@@ -1,19 +1,25 @@
 package com.exallium.todoapp.database
 
 import org.junit.Assert.assertEquals
+import org.junit.Before
 import org.junit.Test
 
 /**
- * Unit testing for IdFactoryImpl
+ * Unit testing for {@link IdFactoryImpl}
  */
 class IdFactoryImplTest {
 
-    val testSubject = IdFactoryImpl()
+    lateinit var testSubject: IdFactoryImpl
+
+    @Before
+    fun setUp() {
+        testSubject = IdFactoryImpl()
+    }
 
     @Test
-    fun createId_when1000Requested_allAreUnique() {
+    fun createId_when10Requested_allAreUnique() {
         // GIVEN
-        val expected = 1000
+        val expected = 10
```

Comment:
Sure.  I will say that this test runs in a couple milliseconds, so I'm not sure it's too big a deal.